wiki/Social_Media_and_its_Negative_Impacts_on_Autonomy.qmd

```diff
@@ -0,0 +1,29 @@
+---
+title: "Social Media and its Negative Impact on Autonomy", Sahebi & Formosa, 2022
+subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+author: "Katharina Finger"
+categories:
+  - "Media Psychology"
+  - "Digital Media"
+  - "Well Being"
+---
+
+**Key Quote** 
+„This led us to focus on the control that social media can have on users’ data, attention and behaviour, which cab result in the disrespect of users’ capacity for autonomy, interference with exercises of their autonomy, and harms to their autonomy competencies.” (S. 20)
+
+**Main Argument** 
+The paper argues that social media influences the autonomy of an individual under the aspects of user data, attention and behaviour. It argues how social media harms the individual autonomy in three ways. First social media platforms disrespect the users’ autonomy by collecting their data without them really knowing and most of them without having the expertise to know better. Second, they put out user-specific content and advertising and therefore interfering episodically and globally with the users’ activity and what they are watching on social media. And lastly, the companies undermine general autonomy competencies like self-esteem and self-determination. It is also argues that users are very often pressured to use big social media platforms in order to connect with their peers or sometimes even for work. 
+
+**Relationship**
+This article is similar to other articles in this field by concentrating mostly on the negative impact of digital power structures on individual autonomy. It uses existing papers and studies to outline common problems which come to mind when thinking about social media and autonomy. It therefore agrees with a lot of studies in this field and acts like a summary.
+
+**Questions**
+Are there any positive effects social media might have on autonomy? 
+What is the perspective from a social media company – is it really their goal to undermine their uers’ autonomy?
+
+**Summary** 

```

Comment:
Can you point out in more detail how autonomy experiences were studied empirically, if at all? If not, can you make some assumptions how this could be reflected in autonomy need experiences as defined in SDT?

Reply:
Autonomy competencies are skills we need to act autonomously, "such as the ability to reason and critically reflect on their values, imagine different alternatives, develop a conception of the good, and regard themselves as self-directing agents worthy of respect" (Sahebi & Formosa, 2022). They may connect to autonomy need experiences because they are the components that stand after the needs. In order to fulfill these needs you need to be able to act them out.
Further research regarding autonomy competencies as a philosophical concept include: Atkins, K. (2006), Autonomy and autonomy competencies: a practical and relational approach. Nursing Philosophy, 7: 205-215. I could not find any empirical research on this topic.